### PR TITLE
Cake 1083 - custom domain for liftigniter calls

### PR DESCRIPTION
**Pull request by nmonterroso**
@Wikia/cake 
https://wikia-inc.atlassian.net/browse/CAKE-1083

use liftigniter through `*.fandommetrics.com`. 

~~DO NOTE MERGE - this is working for me on my devbox, but liftigniter said they'll get back to me with some confirmation from their end.~~

### CODE REVIEW

File: extensions/wikia/Recirculation/js/trackers/liftigniter.js [hunk 1]
```diff
@@ -1,7 +1,19 @@
 (function(w,d,s,p,v,e,r) {w['$igniter_var']=v;w[v]=w[v]||function(){(w[v].q=w[v].q||[]).push(
```
Comment (WikiaTech) [inline extensions/wikia/Recirculation/js/trackers/liftigniter.js:1]: MINOR Reformat the code to have only one statement per line. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AOneStatementPerLine)

File: extensions/wikia/Recirculation/js/trackers/liftigniter.js [hunk 2]
```diff
@@ -1,7 +1,19 @@
...
 arguments)};w[v].l=1*new Date();e=d.createElement(s),r=d.getElementsByTagName(s)[0];e.async=1;
```
Comment (WikiaTech) [inline extensions/wikia/Recirculation/js/trackers/liftigniter.js:2]: MINOR Reformat the code to have only one statement per line. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AOneStatementPerLine)

File: extensions/wikia/Recirculation/js/trackers/liftigniter.js [hunk 3]
```diff
@@ -1,7 +1,19 @@
...
 e.src=p+'?ts='+(+new Date()/3600000|0);
```
Comment (WikiaTech) [inline extensions/wikia/Recirculation/js/trackers/liftigniter.js:3]: MAJOR Remove the use of "|" operator. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3ABitwiseOperators)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 3 issues
* MAJOR 1 major
* MINOR 2 minor

Watch the comments in this conversation to review them.

**Comment by drsnyder** [comment]
LGTM

**Pull request merged**